libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts

```diff
@@ -0,0 +1,221 @@
+import { formatNumber } from '@angular/common';
+
+import { Options } from 'highcharts';
+import * as Highcharts from 'highcharts/highstock';
+
+import { ColorHelper } from '../../../helpers';
+
+export interface HighstockDataPoint {
+  x: number;
+  y: number;
+  id: 'min' | 'max' | '';
+}
+
+export function stockChartOptions(locale: string, height: number) {
```

Comment:
Align with existing chart options - export as const - see https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/components/chart/options/bar.ts

Reply:
Done. It confused Anders as a const, but I'll revert it as such.

Reply:
Done.